frontend/packages/console-app/src/actions/providers/default-provider.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { K8sResourceCommon, referenceFor } from '@console/internal/module/k8s';
+import { useK8sModel } from '@console/shared/src/hooks/useK8sModel';
+import { getCommonResourceActions } from '../creators/common-factory';
+
+export const useDefaultActionsProvider = (resource: K8sResourceCommon) => {
```

Comment:
Is there a TS type we can use for this function to ensure it has the right signature? (args type and return type)

Reply:
There isn't a generic type available but I have added return type for this function. Arg type is already present.